internal/application/application.go

```diff
@@ -419,9 +413,9 @@ func fetch(ctx context.Context, cluster *kubernetes.Cluster, app *models.App) er
 // calculateStatus sets the Status field of the App object.
 // To decide what the status value should be, it combines various pieces of information.
 //- If Status is ApplicationError, leave it as it (it was set by "Lookup")
-//- If there is a pipelinerun, app is: ApplicationStaging
-//- If there is no workload and no pipeline run, app is: ApplicationCreated
-//- If there is no pipelinerun and there is a workload, app is: ApplicationRunning
+//- If there is a staging job, app is: ApplicationStaging --- TODO FIX -- Job done (complete or fail) => Not staging
```

Comment:
These todos look important? But I guess they are fixed?

Reply:
No, it means a fix is to be done. It will also be a more complicated thing (Right now a failed staging blocks attempts at re-staging).

Reply:
See https://github.com/epinio/epinio/issues/1179